internal/local/app.go

```diff
@@ -251,6 +266,32 @@ func (a *App) LoadConfig() error {
 	return nil
 }
 
+// LoadAppMeta will load the local app's data from .mdb/app_meta.json
+func (a *App) LoadAppMeta() error {
+	path := filepath.Join(a.RootDir, FileMdbAppMeta.String())
```

Comment:
just a [naming/nit] ... let's remove the `Mdb` prefix from the `AppMeta` struct and file (if we're being honest to with go patterns, it probably should have been `MDB` anyways)

instead, you can add some `NameMDB`/`NameDotMDB`/`NameHiddenMDB` to `local/data.go` to help "const-ify" the `.mdb` the `".mdb"` string, which could then be included in this `filepath.Join(...)` sequence

Reply:
yeah... this is why I'm not the biggest fan of `app_meta.json` as the filename in the first place... it really doesn't do the purpose of this file justice

however, for all intents an purposes, `local.App` is a different representation of an "app" than `realm.App` is, so I think the point stands here that `local.AppMeta` means different things too (than the `realm` counterpart)

and while I don't think we're adding any other files to `.mdb` anytime soon, that's not a reason to couple directory and filename together like that.  we only have `config.json` in the `sync/` directory but still stick with the pattern of differentiating the directories that the CLI understands and the filenames